scratchclient/Scraping.py

```diff
@@ -402,57 +402,61 @@ def rgb_to_hex(rgb):
 # For lack of a better name
 class ScrapingSession:
     def __init__(self, client):
+        self._headers = {
```

Comment:
Oops, misclicked the "accept" button. I meant for this to be 
```py
self._headers = {
  "User-Agent": "user agent here"
}
```
lmao

Reply:
lmao ok done - btw why formatting for this but not any other line? 👀